app/listeners/webhook_listener.rb

```diff
@@ -51,6 +51,21 @@ def webwidget_triggered(event)
     deliver_webhook_payloads(payload, inbox)
   end
 
+  def contact_created(event)
+    contact = extract_contact_and_account(event)[0]
+    inbox = conversation.inbox
```

Comment:
`conversation` would not be available at this event. Only contact objects would avalable.

Reply:
yeah right contact is created first